backend/battling/views.py

```diff
@@ -33,26 +35,43 @@ def sumValid(pokemon):
     return sum_result
 
 
+def check_valid_team1(curr_form):
+    pokemon1 = get_pokemon_from_api(curr_form.pk11)
+    pokemon2 = get_pokemon_from_api(curr_form.pk12)
+    pokemon3 = get_pokemon_from_api(curr_form.pk13)
+    pokemon1_points = sumValid(pokemon1)
+    pokemon2_points = sumValid(pokemon2)
+    pokemon3_points = sumValid(pokemon3)
+    sum_pokemons_points = pokemon1_points + pokemon2_points + pokemon3_points
+    if sum_pokemons_points <= 600:
+        return True
+    return False
+
+
+def check_valid_team2(curr_form):
+    pokemon1 = get_pokemon_from_api(curr_form.pk21)
+    pokemon2 = get_pokemon_from_api(curr_form.pk22)
+    pokemon3 = get_pokemon_from_api(curr_form.pk23)
+    pokemon1_points = sumValid(pokemon1)
+    pokemon2_points = sumValid(pokemon2)
+    pokemon3_points = sumValid(pokemon3)
+    sum_pokemons_points = pokemon1_points + pokemon2_points + pokemon3_points
+    if sum_pokemons_points <= 600:
+        return True
+    return False
+
```

Comment:
I think it'd be better to merge these 2 functions into a single one (`check_valid_team`) since their behaviors are the same, varying only the analyzed team. You could pass the `pk11`, `pk21`, etc as input parameters and generalize.
```python
def check_valid_team(pk1, pk2, pk3):
    pokemon1 = get_pokemon_from_api(pk1)
    pokemon2 = get_pokemon_from_api(pk2)
    pokemon3 = get_pokemon_from_api(pk3)
    pokemon1_points = sumValid(pokemon1)
    # ...
```